spacy/pipeline/pipes.pyx

```diff
@@ -531,7 +531,7 @@ class Tagger(Pipe):
         if "_SP" in orig_tag_map:
             new_tag_map["_SP"] = orig_tag_map["_SP"]
         cdef Vocab vocab = self.vocab
-        if new_tag_map:
+        if set(new_tag_map) - set(["_SP"]):
```

Comment:
Right, came across this on the new WIP branch for `develop`, didn't realise the same bug would be on `master`. Anyway solved it slightly differently there: https://github.com/explosion/spaCy/blob/whatif/arrow/spacy/pipeline/pipes.pyx#L372 basically do the "_SP" stuff only when you have a `new_tag_map`?

Reply:
That seems simpler, thanks! (I wish this could all be a bit less hacky...)